GerritDownloader/src/main/java/com/holmsted/gerrit/CommandLineParser.java

```diff
@@ -153,6 +161,18 @@ public int getCommitLimit() {
         return limit;
     }
 
+    public String getAfterDate() {
```

Comment:
It'd be better that instead of a string, you'd parse the exception as a Date type and use a custom Converter. That way, you'll also get to use the existing exception handling - which would just cause the app to print usage, in case the args can't be parsed.